referral-form-data-process/serverless.yml

```diff
@@ -4,30 +4,27 @@ provider:
   name: aws
   runtime: nodejs14.x
   region: eu-west-2
+  stage: ${opt:stage}
   lambdaHashingVersion: 20201221
   iam:
     role:
       statements:
         - Effect: "Allow"
           Action:
             - "s3:GetObject"
-          Resource:
-            Fn::Join:
-              - ""
-              - - "arn:aws:s3:::"
-                - "social-care-referrals-bucket"
-                - "/*"
+          Resource: "arn:aws:s3:::${self:custom.referralsBucket}/*"
 
 functions:
   main:
+    name: ${self:service}-${self:provider.stage}
+    handler: referral-form-data-process/main.handler
     environment:
       CLIENT_EMAIL: ${opt:CLIENT_EMAIL}
```

Comment:
Not something that has to change, but are some of these options something that you can know when applying the terraform? If so you can write them to AWS Parameter Store and use serverless SSM access like this: `${ssm:/path/to/stored/value}` for a good example check out our serverless config here: https://github.com/LBHackney-IT/lbh-core-pathway-pilot/blob/013afd371193c2441f113d7de5a7828b4e9b8aa9/serverless.yml#L25

Reply:
🙌🏾 Thanks! 
Yeah, I imagine we can get the names of the bucket and queue as outputs from terraform.
I'll try and refactor this soon!